### DISCUSSION TITLE
Version 9.6.1

### DISCUSSION THREAD

**Discussion by FilipePS** [FilipePS/Traduzir-paginas-web#507] [Announcements]
- Support for older versions of Firefox.
- Updated interface translations.

<hr /><em>This discussion was created from the release <a href='https://github.com/FilipePS/Traduzir-paginas-web/releases/tag/v9.6.1'>Version 9.6.1</a>.</em>

**Comment by Lemon514293676** [comment]
谷歌网页翻译接口用不了了；
Yandex网页翻译接口准确性又低；
可以增加：DeepL网页翻译接口或bing网翻译接口
